src/pages/Selected/Selected.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import SelectForm from './components/SelectForm';
+import SelectSeat from './components/SelectSeat';
+import UserInfo from './components/UserInfo';
+import * as S from './Selected.styles';
+
+export default function Selected() {
+  return (
+    <S.Selected className="inner">
```

Comment:
1차때 scss 쓸 때는 nesting을 위해서 가장 부모 태그의 className을 컴포넌트의 이름과 동일하게 했지만
스타일 컴포넌트에서는 요소 하나하나가 컴포넌트화 되는 부분이기 때문에
컴포넌트의 이름이 겹쳐서 오류를 발생시킬 수도 있습니다!

아래와 같이 가장 부모 되는 스타일 컴포넌트의 이름을 변경해주세요!

```suggestion
export default function Selected() {
  return (
    <S.SelectContainer className="inner">
```